.cicd/scripts/srw_metric_example.sh

```diff
@@ -61,8 +61,6 @@ cd ${workspace}/tests/WE2E
 cd ${workspace}
```

Comment:
In PR #793, the conda environment was changed from `regional_workflow` to `workflow_tools`.  On line 49, please replace `regional_workflow` with `workflow_tools`.

Reply:
Thx for the second eye! Done